src/app/pages/ParaTimesPage/TransactionAmount/FeesSection.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react'
+import { Box, Button, Collapsible, FormField, TextInput } from 'grommet'
+import { useTranslation } from 'react-i18next'
+
+type FeesSectionProps = {
+  feeAmount: string
+  feeGas: string
+  ticker: string
+}
+
+export const FeesSection = ({ feeAmount, feeGas, ticker }: FeesSectionProps) => {
+  const { t } = useTranslation()
+  const [open, setOpen] = useState(false)
+
+  return (
+    <Box pad={{ vertical: 'medium' }}>
+      <Box align="start">
+        <Button
+          label={t('paraTimes.amount.advanced', 'Advanced')}
+          margin={{ bottom: 'medium' }}
+          onClick={() => setOpen(!open)}
+          secondary
+          size="small"
+        />
+      </Box>
+      <Collapsible open={open}>
+        <FormField
+          name="feeAmount"
+          validate={{
+            regexp: /^(\s*|[0-9][0-9]*)$/,
+            message: t('paraTimes.validation.invalidFee', 'Value must be integer greater than or equal to 0'),
+          }}
+        >
+          <TextInput
+            name="feeAmount"
+            placeholder={t('paraTimes.amount.feeAmountPlaceholder', 'Fee Amount (nano {{ticker}})', {
```

Comment:
@pro-wh do we have any explanation of "Fee Amount" vs "Fee Gas" anywhere? Do we name them more-understandably anywhere?

Reply:
Alright, found this in docs:
- https://docs.oasis.dev/oasis-core/consensus/transactions#gas
- https://docs.oasis.dev/general/manage-tokens/advanced/oasis-cli-tools/setup#common-transaction-flags

and they don't mean what thought they did :no_mouth: